examples/metrics/src/main/java/io/opentelemetry/example/metrics/LongCounterExample.java

```diff
@@ -0,0 +1,60 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.metrics.LongCounter;
+import io.opentelemetry.metrics.LongCounter.BoundLongCounter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Tracer;
+import java.io.File;
+import javax.swing.filechooser.FileSystemView;
+
+/**
+ * Example to search all directories for specific file.
+ * Here we will search directory recursively for specific file, using meter to count searched directories.
```

Comment:
I think you missed my point here. The javadoc should explain what the example is trying to teach. We aren't trying to teach people how to traverse directory structures. We're trying to teach people how to use a Counter, a LongCounter in particular.

Reply:
That example trying to show usage of LongCounter, so as i understand u JD will look like
```Example of using LongCounter and BoundCounters to count searched directories.```

Reply:
I would also include a link to the relevant specification, and a link to the javadoc for LongCounter class itself, if possible.

Reply:
Applied suggestions, also added span status and catch block